packages/text-annotator/test/index.html

```diff
@@ -287,12 +287,13 @@ <h3>Not annotatable block!</h3>
       window.onload = async () => {
         var contentContainer = document.getElementById('content');
 
-        const style = ((annotation, state, z) => ({
-          fillOpacity: 0.2,
-          underlineColor: '#00ff00',
-          underlineOffset: z * 3.5,
-          underlineThickness: 2
-        }));
+        const style = ((annotation, state, z) =>
+	        ({
+		        fillOpacity: 0.2,
+		        underlineColor: '#9b3c06',
```

Comment:
Made the color a bit less acidic: 
![image](https://github.com/recogito/text-annotator-js/assets/68850090/88be3869-1f72-48ca-976e-8a30aea34c67)